RangeHTTPServer/__main__.py

```diff
@@ -25,6 +25,9 @@
 parser.add_argument('port', action='store',
                     default=8000, type=int,
                     nargs='?', help='Specify alternate port [default: 8000]')
+parser.add_argument('-b', '--bind', metavar='ADDRESS',
+                        help='Specify IP address'
```

Comment:
nit: move help text onto one line and add a space before the `[`.

When I run `python -m http.server --help` I see this:

```
  -b ADDRESS, --bind ADDRESS
                        bind to this address (default: all interfaces)
```

Is "all interfaces" actually the default, rather than "localhost"?